code/game/area/Space Station 13 areas.dm

```diff
@@ -816,6 +817,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "\improper Construction Area"
 	icon_state = "construction"
 
+/area/hallway/secondary/entry
+	forbid_events = TRUE
+

```

Comment:
Hey, Raz? Ya wanted events to not make grubs spawn at the docks? Looks like this is it. 👀

Reply:
It already works fine